src/Command/PullRequest/PullRequestPatOnTheBackCommand.php

```diff
@@ -48,11 +81,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $adapter = $this->getAdapter();
         $pr = $adapter->getPullRequest($prNumber);
+        if (!$pat = $input->getOption('pat')) {
+            $pat = null;
```

Comment:
The `if` and this assignment seems a bit redundant, since `$input->getOption('pat')` would anyway return `null` if no option is provided